core/src/main/scala/filodb.core/store/package.scala

```diff
@@ -145,8 +146,9 @@ package object store {
     def scanRows(dataset: Dataset,
                  columnIDs: Seq[ColumnId],
                  partMethod: PartitionScanMethod,
-                 chunkMethod: ChunkScanMethod = AllChunkScan): Iterator[RowReader] =
-      source.scanPartitions(dataset.ref, columnIDs, partMethod, chunkMethod)
+                 chunkMethod: ChunkScanMethod = AllChunkScan,
+                 querySession: QuerySession = QuerySession.forTestingOnly): Iterator[RowReader] =
```

Comment:
Should the scanRows method be tagged as "for testing only" too?

Reply:
Yes - it actually should. Dont know the original intent. Will just leave it.

Reply:
I think a comment will help. I see "forTestingOnly" in the code and this isn't mirrored in the scanPartitions doc or method name.